django_tasks/backends/database/backend.py

```diff
@@ -43,6 +46,17 @@ def _task_to_db_task(
             backend_name=self.alias,
         )
 
+        # sign value without timestamp
+        # TimestampSigner doesn't support setting our own timestamp?
+        if settings.TASKS.get(self.alias, {}).get("SIGN_TASKS") is True:
+            signer = Signer()
```

Comment:
Issue: The signature should be salted. For compatibility, we should also specify the `algorithm`, as this could change in future Django versions, leading to signature issues.